packages/commons/src/context/context.ts

```diff
@@ -21,7 +22,9 @@ const globalStore = new AsyncLocalStorage<AppContext>();
 const defaultAppContext: AppContext = {
   authData: {
     userId: "",
-    organizationId: "",
+    // TODO: this is a workaround to avoid to change the type
```

Comment:
Will this be addressed in this PR / tracked on Jira?

Reply:
Not tracked and not addressed. I don't have a clear solution here, and I don't even know if it's so important to resolve this issue. Could this situation happen? @Viktor-K what do you think?

Reply:
The workaround it's fine. I removed the TODO comment